regular_expression/1/1.py

```diff
@@ -0,0 +1,11 @@
+import re
+with open('references.txt') as input:
+    text = input.read()
+pattern = re.compile(r'ftp[\w./]*\.[\w./]*[;\s]')
```

Comment:
Можно было в паттерне заключить ссылку в скобки, чтобы findall вернул только её, в таком случае не пришлось бы потом удалять лишние символы.
Примерно вот так
```suggestion
pattern = re.compile(r'(ftp[\w./]*\.[\w./]*)[;\s]')
```